src/main/java/io/leonis/torch/component/CenteredLabel.java

```diff
@@ -0,0 +1,51 @@
+package io.leonis.torch.component;
+
+import com.googlecode.lanterna.*;
+import com.googlecode.lanterna.gui2.*;
+import java.util.*;
+import java.util.stream.IntStream;
+import lombok.*;
+
+/**
+ * The Class CenteredLabel.
```

Comment:
![](https://media.giphy.com/media/10uct1aSFT7QiY/giphy.gif)

Reply:
This is a thing we do everywhere in the codebase. Might want to create an issue or discuss this on Slack if you want to change it.